src/components/sbb-menu/sbb-menu.scss

```
@@ -118,16 +125,16 @@
   }
 
   @include sbb.mq($from: medium) {
-    top: 0;
-    left: 0;
-    right: unset;
-    bottom: unset;
+    inset-block-start: 0;
+    inset-inline-start: 0;
+    inset-inline-end: unset;
+    inset-block-end: unset;
```

Nitpick; Would this be equal?
```suggestion
    inset: 0 unset unset 0;
```

---

Unfortunately, this shorthand would not work because apparently 'unset' is not supported there. Where possible I replaced it with something like `inset: 0 auto auto 0`, so I can try using this but I'm not sure if it has the same effect.